.github/workflows/update-browser-version.yml

```diff
@@ -29,6 +29,10 @@ jobs:
         uses: peter-evans/create-pull-request@v5
         with:
           token: ${{ secrets.BROWSER_AUTOMATION_BOT_TOKEN }}
-          commit-message: Update the browser version
+          branch: create-pull-request/browser-update
```

Comment:
I'd put the name of the bot here: `<bot>/<workflow name>`, so: `browser-automation-bot/update-browser-version`.

Reply:
This matches our convention to use the author as the first branch name part